internal/pkg/template/templates/environment/cf.yml

```diff
@@ -41,6 +41,9 @@ Parameters:
 Conditions:
   CreateALB:
     !Not [!Equals [ !Ref ALBWorkloads, "" ]]
+  CreateCerts: !And
```

Comment:
Ah, I think I missed something important - in an environment, it is possible for 1 lbws to import an alb, and the other lbws to not import an alb. In this case, `CreateALB` will be `true`, and we will start upserting aliases records for that LBWS with import alb - that users are managing themselves.

What do you think about not passing the `Aliases` field to env controller from lbws: https://github.com/aws/copilot-cli/blob/8491d15c70d53548ff310b330962d93a40fbfd30/internal/pkg/template/templates/workloads/partials/cf/env-controller.yml#L9


@iamhopaul123 curious about your thoughts too - because this is similar to imported cert, but not exactly the same (imported cert is env-level, imported alb is svc-level)

Reply:
That's a very good point. We probably do not want to pass the `alias` field if an LBWS use imported ALB 👍

Reply:
So we shouldn't pass the `Aliases` field but we should also not pass the params (`Parameters: [Aliases]`), right?

Reply:
> but we should also not pass the params (Parameters: [Aliases]), right?

which lines of codes does this refer to?

Reply:
see the latest commit :)